src/app/components/tile/index.html

```diff
@@ -175,4 +175,56 @@
     </sky-demo-page-code>
   </sky-demo-page-example>
 
+  <sky-demo-page-content sectionHeading="UX guidelines">
+    <p>
+      Tiles are containers to organize varied content. They divide pages into multiple categories and group related content. Use tiles to provide access to a wide range of content where users benefit from the ability to reorder, expand and collapse, and personalize tiles. For more details, see <a routerLink="/design/guidelines/content-containers">content container guidelines</a>.
+    </p>
+
+  </sky-demo-page-content>
+
+  <stache-row>
+    <stache-column screenSmall="6">
+      <div class="sky-subsection-heading">Usage</div>
+      <p>
+        Use tiles to divide pages with many categories of content, especially when the format or presentation differs across categories.
```

Comment:
The other major factor here is whether the contents need to be reordered. @Blackbaud-AdamFunderburk is the guideline here to use tiles for differing presentations even if there is no need for the user to rearrange? I've generally viewed the need to rearrange as the first check for tiles vs page sections.

Reply:
Yeah, the mention of differing presentations might be a red herring here. It's more important to make the point of
"... especially when the content is varied enough that users would benefit from the capability to reorder or expand/collapse the categories."

Reply:
Updated with your verbiage, Adam